Organic/AdsTxt.php

```diff
@@ -22,24 +23,18 @@ public function __construct( Organic $organic ) {
     }
 
     public function get() {
```

Comment:
I think we should rename the method considering that it just returns URL where to redirect

Reply:
Agreed - will rename.